cmd/kubeadm/app/cmd/phases/upgrade/node/controlplane.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package node
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+
+	"k8s.io/kubernetes/cmd/kubeadm/app/cmd/options"
+	"k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases/workflow"
+	"k8s.io/kubernetes/cmd/kubeadm/app/phases/upgrade"
+	"k8s.io/kubernetes/cmd/kubeadm/app/util/apiclient"
+)
+
+// NewControlPlane creates a kubeadm workflow phase that implements handling of control-plane upgrade.
+func NewControlPlane() workflow.Phase {
+	phase := workflow.Phase{
+		Name:  "control-plane",
```

Comment:
`kubeadm init phase control-plane` does not provide an alias, so I don't think we should add one here.
Eventually, this can be done in a follow up PR, but it should be done consistently across all control-plane commands

Reply:
our naming has the `-` between words everywhere.